### PR TITLE
[NONMODULAR] [READY] Cargonia bad

### PR DESCRIPTION
**Pull request by lyricalpaws**
## About The Pull Request
Removes the mosins from the russian surplus crate, because its only ever bought by cargonian tiders that want to larp as russians.

## Why It's Good For The Game
Less weapons for cargo, less lrp tiding, better for everyone (sec never uses mosins anyway)

## Changelog
:cl:
balance: Removed bolt action mosin rifles from the russian surplus crate. 
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -626,9 +626,8 @@
 					/obj/item/clothing/under/costume/soviet,
 					/obj/item/clothing/mask/russian_balaclava,
 					/obj/item/clothing/head/helmet/rus_ushanka,
-					/obj/item/clothing/suit/armor/vest/russian_coat,
-					/obj/item/gun/ballistic/rifle/boltaction,
-					/obj/item/gun/ballistic/rifle/boltaction)
+					/obj/item/clothing/suit/armor/vest/russian_coat)
+					//SKYRAT EDIT: REMOVAL - Removed /obj/item/gun/ballistic/rifle/boltaction
```
Comment (tf-4) [inline code/modules/cargo/packs.dm]: probably should lower the cost of the crate a decent amount considering its main appeal is gone

Comment (lyricalpaws) [reply code/modules/cargo/packs.dm]: Halfed it

**Comment by lyricalpaws** [comment]
Finally, all checks have passed

**Comment by Stalkeros2** [comment]
(

**Comment by hugmeorelse** [comment]
Someone needs to change the github-actions bot's aggressiveness on staleness.

**Pull request closed without merging**